src/Tracker/trackerData.tsx

```diff
@@ -503,4 +503,250 @@ export const analyticsEventsMapping = new Map<
+      },
+    },
+  ],
+  [
+    'screen_document_front_capture_camera_not_working_no_fallback',
+    {
+      eventName: 'DOCUMENT_CAPTURE_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
+        ui_alerts: { camera_not_working_no_fallback: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_back_capture_camera_not_working_no_fallback',
+    {
+      eventName: 'DOCUMENT_CAPTURE_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'back',
+        ui_alerts: { camera_not_working_no_fallback: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_face_selfie_capture_camera_not_working_no_fallback',
+    {
+      eventName: 'FACE_SELFIE_CAPTURE_ERROR',
+      properties: {
+        event_type: 'view',
+        ui_alerts: { camera_not_working_no_fallback: 'error' },
+      },
+    },
+  ],
+
+  [
+    'screen_face_face_video_capture_camera_inactive',
+    {
+      eventName: 'FACE_VIDEO_CAPTURE_ERROR',
+      properties: {
+        event_type: 'view',
+        ui_alerts: { camera_inactive: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_front_capture_camera_inactive',
+    {
+      eventName: 'DOCUMENT_CAPTURE_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
+        ui_alerts: { camera_inactive: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_back_capture_camera_inactive',
+    {
+      eventName: 'DOCUMENT_CAPTURE_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'back',
+        ui_alerts: { camera_inactive: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_face_selfie_capture_camera_inactive',
+    {
+      eventName: 'FACE_SELFIE_CAPTURE_ERROR',
+      properties: {
+        event_type: 'view',
+        ui_alerts: { camera_inactive: 'error' },
+      },
+    },
+  ],
+
+  [
+    'screen_face_face_video_camera_inactive_no_fallback',
+    {
+      eventName: 'FACE_VIDEO_CAPTURE_ERROR',
+      properties: {
+        event_type: 'view',
+        ui_alerts: { camera_inactive_no_fallback: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_front_capture_camera_inactive_no_fallback',
+    {
+      eventName: 'DOCUMENT_CAPTURE_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'front',
+        ui_alerts: { camera_inactive_no_fallback: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_document_back_capture_camera_inactive_no_fallback',
+    {
+      eventName: 'DOCUMENT_CAPTURE_ERROR',
+      properties: {
+        event_type: 'view',
+        document_side: 'back',
+        ui_alerts: { camera_inactive_no_fallback: 'error' },
+      },
+    },
+  ],
+  [
+    'screen_face_selfie_capture_camera_inactive_no_fallback',
+    {
+      eventName: 'FACE_SELFIE_CAPTURE_ERROR',
+      properties: {
+        event_type: 'view',
+        ui_alerts: { camera_inactive_no_fallback: 'error' },
```

Comment:
I think this might be a "warning". When they have the blue background they are warning. If it comes from the function `getInactiveError` then it's a warning

Reply:
Thanks!